src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<p>This file is not copied by the <code>fastoad notebooks</code> command because there is no <code>__init__.py</code> files in <code>working_folder </code>and its subfolders.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Changed in 3a141cc1